bundles/org.openhab.binding.rotel/src/main/java/org/openhab/binding/rotel/internal/handler/RotelHandler.java

```diff
@@ -1001,9 +1158,16 @@ private void handleToneCmd(int current, String channel, Command command, int nbS
      * @throws InterruptedException in case of interruption during a thread sleep
      */
     private void selectToneControl(int nbSelect) throws RotelException, InterruptedException {
-        // No tone control select command for RSX-1065
-        if (connector.getProtocol() == RotelProtocol.HEX && connector.getModel() != RotelModel.RSX1065) {
-            selectFeature(nbSelect, RotelCommand.RECORD_FONCTION_SELECT, RotelCommand.TONE_CONTROL_SELECT);
+        switch (connector.getModel().getName()) {
```

Comment:
No need to change that as your model is using ASCII protocol. 
This was covered by the first condition in the if.

Reply:
ok: change re-changed